stepflow-rs/crates/stepflow-main/src/cli.rs

```diff
@@ -142,6 +143,27 @@ pub enum Command {
         #[command(flatten)]
         config_args: ConfigArgs,
     },
+    /// Validate workflow files and configuration.
+    Validate {
+        /// Path to the workflow file to validate.
+        #[arg(
+            long = "flow",
+            value_name = "FILE",
+            value_hint = clap::ValueHint::FilePath
+        )]
+        flow_path: Option<PathBuf>,
+
+        #[command(flatten)]
+        config_args: ConfigArgs,
+
+        /// Only validate the configuration file, not the workflow.
+        #[arg(long)]
+        config_only: bool,
```

Comment:
Probably doesn't matter, but there is a way to make these mutually exclusive using the annotations (I don't remember it off the top of my head).